lib/top_row.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+import 'save_movie_button.dart';
+import 'stars.dart';
+import 'text_styles.dart';
+import 'like_counter.dart';
+import 'movie.dart';
+
+/*
+* Represents the row at the top right corner, between the background and the poster image.
+* It works as a container for the save button, like button, and the rating stats.
+* */
+class TopRow extends StatelessWidget {
+  const TopRow({
+    super.key,
+    required this.movie,
+  });
+
+  static const double posterContainerWidth = 170;
+  static const double posterContainerHeight = 230;
+  static const double buttonsTopPadding = 80;
+  static const double buttonsLeftPadding = 15;
+  final Movie movie;
+
+  static const BoxShadow posterBoxShadow = BoxShadow(
+    blurRadius: 18,
+    spreadRadius: 0.75,
+    offset: Offset.zero,
+    blurStyle: BlurStyle.normal,
+  );
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        Container(
+          width: posterContainerWidth,
+          height: posterContainerHeight,
+          decoration: BoxDecoration(
+            boxShadow: const [posterBoxShadow],
+            image: DecorationImage(
+              image: Image.network(movie.poster).image,
```

Comment:
This could be replaced with `NetworkImage(movie.poster)` object!

Reply:
Thanks, done!